dgea/plot_dgea.py

```diff
@@ -94,16 +94,30 @@ def plot_volcano():
         
         df_plot = deseq_results.copy()
         df_plot["-log10(padj)"] = -np.log10(df_plot["padj"])
+        min_value = 1e-100
+        df_plot["p-value"] = df_plot["padj"].apply(lambda x: x if x > min_value else f"less than {min_value}")
```

Comment:
Very sleek implementation. I think we can set the `min_value` higher, since the current one would probably look like `0.00000000.....00001`. `1e-5` should be sufficient

Reply:
Yes we could, but isn't it nice to have a look at the exact p-values of outliers if it is possible?

Reply:
The problem is that even if we show the exact values, nobody will count the zeros. We could show the scientif notation instead, which would make this easier. A guide for how to achieve this can be found [here](https://stackoverflow.com/questions/6913532/display-a-decimal-in-scientific-notation)